README.md

```diff
@@ -21,3 +21,28 @@ A multiarch [tang] Docker image, based on [Alpine Linux].
 
 [Alpine Linux]: https://alpinelinux.org/
 [tang]:         https://github.com/latchset/tang
+
+
+# Quick-start
+To start a one-shot container simply run
+`docker run -it --rm -p 8080:8080/tcp padhihomelab/tang`
+where the standard server port, `8080`, will be exposed on your host machine.
+
+## Persist data
+The "quick-start command" will not persist its data (located at `/data`) beyond the lifetime of the container.  
+To link the tang server's data with a persistent volume on the host run:  
+`docker run -it --rm -p 8080:8080 --volume "/home/myUser/whatever":"/data" padhihomelab/tang`  
+or if you want to work with docker volumes:  
+``` sh
+#!/bin/sh
+data_volume=$(docker volume create)
+docker run -it --rm -p 8080:8080 --volume "$data_volume":"/data" padhihomelab/tang
+```
```

Comment:
Let's remove this part about docker volumes since it just creates a volume with a random name.

The initial command you mentioned: `docker run -it --rm -p 8080:8080/tcp padhihomelab/tang` creates an _anonymous_ volume and the user can peek into the specific volume being used by his/her container using `docker inspect`.

Reply:
I removed the example, where one explicitly creates docker volumes.
Instead I explained how to reach the implicitly (auto-generated) docker volume.
Probably that's cool for people that quick-start a container and later try to find the generated private keys from the server.